CIP-1694/README.md

```diff
@@ -293,6 +301,10 @@ Each vote transaction consists of the following:
 * a yes/no/abstain vote.
 
 Note that "abstained" votes are not included in the "active voting stake".
+Note also that a vote to abstain is different than abstaining from voting
+(the former is visible on chain, the latter is not).
+To avoid confusion, we will only use the word "abstain" from this point onward to mean a vote
```

Comment:
Why not us it to mean either? Isn't that what you want?

Reply:
See my other comment, "blank vote" vs "abstain"